tests/fixtures/cli.py

```diff
@@ -24,8 +24,9 @@ def __init__(self, *args, snowplow: SnowplowMicro | None = None, **kwargs):
 
     def invoke(self, *args, **kwargs) -> Any:
         results = super().invoke(*args, **kwargs)
-        if self.snowplow:
-            assert self.snowplow.all()["bad"] == 0
+        if self.snowplow:  # nocov
```

Comment:
I can't find `nocov` in the coverage docs, does it work the same as `# pragma: no cover`?

https://coverage.readthedocs.io/en/6.3.3/excluding.html#excluding-code-from-coverage-py

Reply:
Ah I just saw you'd added to the coverage config, sorry about that. `pragma: no cover` is built-in so it's probably preferred anyway 👍